editions/test/tiddlers/tests/test-wikitext-tabs-macro.js

```diff
@@ -0,0 +1,43 @@
+/*\
+title: test-wikitext-tabs-macro.js
+type: application/javascript
+tags: [[$:/tags/test-spec]]
+
+Tests the tabs-macro, so there is _no_difference in the HTML structure between the existing implementation and 
```

Comment:
Tests the core tabs macro by comparing the HTML output with a stored template. Intended to permit future readability improvements.

Reply:
ok